examples/apps/aml/workload/aml_logic.cpp

```diff
@@ -0,0 +1,345 @@
+/* Copyright 2019 Banco Santander S.A.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "aml_logic.h"
+
+AmlResultLogic::AmlResultLogic() {}
+
+AmlResultLogic::~AmlResultLogic() {}
+
+template<typename Out>
+
+void splot(const std::string &str, char delim, Out result)
+{
+    std::size_t current, previous = 0;
+
+    current = str.find(delim);
+    while (current != std::string::npos) 
+    {
+        std::string item = str.substr(previous, current - previous);
+        if (item.compare("") != 0)
+            *(result++) = item; 
+        previous = current + 1;
+        current = str.find(delim, previous);
+    }
+    
+    std::string item = str.substr(previous, current - previous);
+    if (item.compare("") != 0)
+        *(result++) = item;
+}
+
+std::vector<std::string> splot(const std::string &s, char delim)
+{
+    std::vector<std::string> elems;
+    splot(s, delim, std::back_inserter(elems));
+    return elems;
+}
+
+std::string AmlResultLogic::getNameFromId(std::string id) {
+    std::string country = id.substr(0,2); 
+    int i;
+    for(i = 0; i < numberTotalCountries; i++){
```

Comment:
For here and everywhere else add a space after `for` and `if`.
That is change `for(` to `for (` and `if(` to `if (`